### DISCUSSION TITLE
How to think about performance bottleneck, when to expand?

### DISCUSSION THREAD

**Discussion by BlackWand1** [minio/minio#18902] [Q&A]
Hello,

We have a pool with 4 servers, each with 12 drives. Even now when we are at maybe 200-400 Mbit incomming data all drives is about 60-100ms in write and read delay. Do we have to much in-data compared to the underlying IO performance or is 60-100ms not much?

Is there any good pointer where one should consider expand with more drives and such?
At what number of xxxx ms will the system be more or less unresponsive?

**Comment by BlackWand1** [comment]
Just to clarify, it is 4x12 normal spinning disk drives.

**Reply by klauspost** [reply]
We do not provide performance analysis outside our [subscriptions](https://min.io/pricing).
